### ISSUE TITLE
Task - Browse Collection Objects - UI Issues

### ISSUE THREAD

**Issue by Gerutha** [SpeciesFileGroup/taxonworks#3228]
### Feature or enhancement

1. Not all values under the Collecting Event panel on the CO page are editable, cached_level0_geographic_name for example. It should be visually clear which fields are editable. 
2. CO fields (currently displayed in the Darwin Core panel) can only be changed by going through the edit interaction. **Fix:** CO fields should be directly editable (just like the CE values in the Collecting Event panel).
3. Navigating to edit the CO brings you to the Comprehensive Specimen Digitization page. This page shows a lot of information about the CE, and allows you to edit CE information as well as CO information. This is overload, as the user is simply interested in editing the CO information. **Fix:** Implement a page that focuses solely on editing the CO information, while still allowing the user to view the CE information if desired.
4. Under the Timeline panel, clicking on 'Given identifier'/catalog number takes you to another Collection objects page with Previous/Next buttons. (see https://sandwich.taxonworks.org/collection_objects/1932778). This is different from the display you get with Browse CO (https://sandwich.taxonworks.org/tasks/collection_objects/browse?collection_object_id=1932778). So there are more than one type of page display used for a single CO. **Fix:** Whether browsing, filtering or through the Data lists, the link should take you to the same CO page.
5. Word wrap issues even with full screen view (see screenshot below). Wrapping prevents the information from being understood easily. It is difficult to read just two or three characters per line. There is so much unused space that could mitigate the need for wrapping.
Wrapping in Collecting Event panel on verbatim_locality and verbatim_collectors fields (if too long). **Fix:** remove/hide unused or empty panels to make data panels larger
Extreme wrapping in Identifiers panel under “On” column. **Fix:** not sure the purpose of this panel. Has duplicate data or unnecessary data, e.g., cf235620-40de-4886-8997-d5b8094a8a91

### Location

Task - Browse Collection Objects

### Screenshot, napkin sketch of interface, or conceptual description

![CO wrap](https://user-images.githubusercontent.com/74991451/207161495-765a9617-5c95-463a-a2c3-bc554864ed10.jpg)

**Comment by Gerutha** [comment]
@bpescador, @cgrinter, @rich-keller

**Comment by AntWeb-org** [comment]
No problem - there is a lot that we'll be submitting but I will clarify when I can.

Right now editing from BCO takes you to the CSD where all CO, CE, geographic area and label data lives, whereas, editing from BCE takes you to an Edit CE page where you can make edits only to the CE data.

If BCO took you to an ‘Edit CO’ page to only edit CO data what would be great. But for now certain fields could be directly editable on the BCO page such as Catalog number or identifier, namespace, repository, Preparation, OTU, determiner, det. date, trip code, life stage/caste, secondary repository (located at), type status, Mol Project Notes, and additional CO notes. Basically, swapping the Timeline and CE panels and adding some CO fields.

Thinking about it further, I’m not sure if having all CE data editable on a BCO page is a good idea. The CE data links to other COs and changing/updating parts of the CE data for one CO will affect all other COs associated with that CE. Maybe this should be separate ticket.

**Comment by mjy** [comment]
> Thinking about it further, I’m not sure if having all CE data editable on a BCO page is a good idea. The CE data links to other COs and changing/updating parts of the CE data for one CO will affect all other COs associated with that CE. Maybe this should be separate ticket.

Right. We are experimenting with this right now.  Note that when you click an editable (as you note, not all) CE field in browse CO it will tell you how many records are affected.  We did this anticipating you might want to change them from there, it's definitely an experiment though.

**Comment by mjy** [comment]
> But for now certain fields could be directly editable on the BCO page such as Catalog number or identifier, namespace, repository, Preparation, OTU, determiner, det. date, trip code, life stage/caste, secondary repository (located at), type status, Mol Project Notes, and additional CO notes. Basically, swapping the Timeline and CE panels and adding some CO fields.

I feel this is not all going to happen in Browse CO. We need a new, stripped down task, that might act a little like browse CO in some ways, and like Comprehensive in others, but be focused on driving changes, rather than adding new data.

**Comment by rich-keller** [comment]
Just my 2 cents, but it seems to me that the way to go is to provide more UI customization capabilities in the base Browse CO task, so that projects can configure the UI so it meets their needs. Otherwise, you will need to create "stripped down" tasks for every new project that comes along. And there is already a proliferation of somewhat similar tasks floating around.

**Comment by mjy** [comment]
@rich-keller we'll focus that discussion here: #3244.

**Comment by AntWeb-org** [comment]
requires UI customization
